uamqp_generator/amqp_definitions_c.cs

```diff
@@ -19,8 +19,8 @@ namespace amqplib_generator
     /// Class to produce the template output
     /// </summary>
     
-    #line 1 "g:\repos\azure-event-hubs-c\deps\azure-uamqp-c\uamqp_generator\amqp_definitions_c.tt"
-    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("Microsoft.VisualStudio.TextTemplating", "15.0.0.0")]
+    #line 1 "C:\code\s1\azure-uamqp-c\uamqp_generator\amqp_definitions_c.tt"
```

Comment:
Did this get changed to your local path?

Reply:
Yes, that's normal for the amqp_definitions. Whoever updates it most recently gets those paths updated to their local paths. The VS text-template engine does that automatically.